src/components/HomePage/HomePage.module.scss

```diff
@@ -0,0 +1,162 @@
+@use '../../styles/colors.scss' as color;
+@use '../../styles/fonts.scss' as font;
+@use '../../styles/mediaQueries.module.scss' as mQ;
+
+.container {
+  display: flex;
+  flex-direction: column;
+  padding-inline: 16px;
+  background-color: color.$background-color;
+
+  @include mQ.tablet {
+    padding-inline: 24px;
+  }
+
+  @include mQ.desktop {
+    padding-inline: 160px;
+    padding-top: 45px;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-start;
+  }
+
+  .titleContainer {
+    display: flex;
+    flex-direction: column;
+    gap: 12px;
+    margin-bottom: 20px;
+
+    @include mQ.tablet {
+      gap: 20px;
+      margin-bottom: 24px;
+    }
+
+    @include mQ.desktop {
+      gap: 24px;
+      margin-bottom: 63px;
+    }
+
+    .title {
+      color: color.$black;
+      @include font.h5-font;
+      z-index: 1;
+      padding-top: 32px;
+
+      @include mQ.tablet {
+        @include font.h2-font;
+        font-weight: bold;
+        padding-top: 40px;
+      }
+
+      @include mQ.desktop {
+        width: 453px;
+        padding-top: 60px;
+      }
+
+      .subtitle {
+        position: relative;
+        display: inline-block;
+
+        &::after {
+          content: '';
+          position: absolute;
+          left: 49%;
+          bottom: 0;
+          width: 98%;
+          height: 22px;
+          background: rgba(36, 101, 187, 0.3);
+          transform: translateX(-50%);
+          z-index: -1;
+
+          @include mQ.tablet {
+            height: 42px;
+          }
+        }
+      }
+    }
+
+    .paragraph {
+      @include font.description-text;
+      color: color.$gray-dark;
+    }
+  }
+
+  .containerLinks {
+    padding: 46px 16px 20px 16px;
+    display: flex;
+    justify-content: space-between;
+    width: 343px;
+    order: 2;
+
+    @include mQ.tablet {
+      width: 262px;
+      left: 160px;
+      align-self: flex-start;
+      gap: 50px;
+      padding-bottom: 44px;
+      padding-top: 8px;
+    }
+
+    @include mQ.desktop {
+      padding-top: 240px;
+      padding-bottom: 64px;
+      left: 160px;
+      align-self: flex-start;
+    }
+
+    .link {
+      @include font.body-text;
+      text-decoration: none;
+      color: color.$black;
+      border-bottom: 1px solid black;
+      &:hover {
+        transition: width 0.3s ease;
+        border-bottom: 1px solid color.$gray-dark;
+      }
+    }
+  }
+
+  .img {
+    max-width: 921px;
+    max-height: 602px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    left: 0;
+    order: 2;
+
+    @include mQ.tablet {
+      position: relative;
+      left: 0;
+      width: calc(100% + 182px);
+    }
+
+    @include mQ.desktop {
+      margin-left: 24px;
+      order: 3;
+      position: absolute;
+      left: 560px;
+      width: calc(150% - 102px);
+    }
+  }
```

Comment:
There is an issue with background begin too small or content too large
<img width="586" alt="Zrzut ekranu 2024-07-25 o 15 26 55" src="https://github.com/user-attachments/assets/e40c445b-f7c9-4280-98fc-7158b074dd89">